### ISSUE TITLE
`ar.exe` tries to load libdep.a (from lib\bfd-plugins) as a DLL and fails

### ISSUE THREAD

**Issue by reznikmm** [alire-project/GNAT-FSF-builds#25]
If I build

    gprbuild -P langkit_support.gpr -p

on running `ar.exe` a error dialog appears with a message like 
> `C:\alire\gnat_native_11.2.2_b741ebbc\lib\bfd-plugins\libdep.a` is not a windows program and can't be run

When I press `Ok` button it works after all.

Files are:
* `langkit_support.ads`
```
package Langkit_Support with Pure is
end Langkit_Support;
```

* `langkit_support.gpr`
```
library project Langkit_Support is

   for Source_Dirs use (".");
   for Object_Dir use "obj/";

   for Library_Name use "langkit_support";
   for Library_Kind use "static";
   for Library_Interface use
     ("Langkit_Support");

   for Library_Dir use "lib/";

end Langkit_Support;
```

**Comment by Fabien-Chouteau** [comment]
I also found this: https://github.com/msys2/MINGW-packages/blob/68b22c4a024bf434635441c6ce5ffc3e7cf8a220/mingw-w64-binutils/PKGBUILD#L134

**Comment by Fabien-Chouteau** [comment]
Fixed in #26
